draft-ietf-quic-transport.md

```diff
@@ -1050,8 +1050,14 @@ An endpoint SHOULD ensure that its peer has a sufficient number of available and
 unused connection IDs.  Endpoints store received connection IDs for future use
 and advertise the number of connection IDs they are willing to store with the
 active_connection_id_limit transport parameter.  An endpoint MUST NOT provide
-more connection IDs than the peer's limit.  An endpoint that receives more
-connection IDs than its advertised active_connection_id_limit MUST close the
+more connection IDs than the peer's limit, with the exception that connection
```

Comment:
Both these sentences are onerous to read and super long... can you break them up a bit?